bench/BenchAll.hs

```diff
@@ -65,7 +80,7 @@ countToZero n = Just (n, n - 1)
 
 -- | Few-enough repetitions to avoid making GC too expensive.
 nRepl :: Int
-nRepl = 10000
+nRepl = 100000
```

Comment:
This is just to reduce noise and allocation overheads, right? I have some work elsewhere that should hopefully improve this situation without needing to increase this value; I can separate it into its own PR tomorrow.